Update User Stories (#93)
BarbzCodez/Spendr: documentation/project-proposal.md

### Budget Management

> 1. As a user, I want to define weekly, monthly, and annual budgets for specific categories so that I can set spending limits for different time frames.\

Reviewer: ```suggestion
1. As a user, I want to define a budget with frequency (weekly/monthly/annual) and a category so that I can set spending limits for different time frames and categories.\
```

When I select to remove an existing budget,\
Then the budget should be successfully removed and not show on the budget tab.

4. As a user, I want to see how much of my budget I have used so that I can adjust my spending habits.\
**Acceptance Criteria:**\
Given I am a logged-in user,\
When I navigate to the budget tab,\

> Then I should see already existing budget and their progress.

Reviewer: ```suggestion
Then I should see already existing budgets and their progress.
```

### Expense Analytics

> 1. As a user, I want visual insights into my expenses from the first day of the month to the current day of the month so that I can quickly understand my spending habits.\

Reviewer: ```suggestion
1. As a user, I want visual insights into my daily expenses from the first day of the month to the current day of the month so that I can quickly understand my spending habits.\
```

**Acceptance Criteria:**\
Given I am a logged-in user,\
When I access the app's analytics dashboard,\
Then I should be able to see spending trends and patterns displayed through graphs and charts.

> 2. As a user, I want to compare expenses over two separate past months so that I can identify trends or changes in my spending.\

Reviewer: ```suggestion
2. As a user, I want to visually compare expenses over two separated past months so that I can identify trends or changes in my spending.\
```

Author: I dont think it should be separated (?)

**Acceptance Criteria:**\
Given I am a logged-in user,\
When I choose the "Compare" option in the analytics dashboard,\
Then I should be able to see spending trends and habits of two past months side by side.

> 3. As a user, I want to see spending patterns for the current month each category so that I can gain insights into where most of my money goes.\

Reviewer: ```suggestion
3. As a user, I want to see spending patterns of each category for the current month so that I can gain insights into where most of my money goes.\
```

Author: of each category in the current month sounds better?

**Acceptance Criteria:**\
Given I am a logged-in user,\
When I chose to filter the analytics dashboard by category,\
Then then analytics dashboard should correctly update and I should be able to see the spending patterns for the chosen category.

4. As a user, I want a text summary for changes while comparing expenses so that I can be promptly informed of significant variations in my spending habits.\
**Acceptance Criteria:**\
Given I am a logged-in user,\

> When Im comparing month expenses,\

Reviewer: ```suggestion
When I am comparing expenses of 2 past months,\
```

> Then I should be able to see an text informing me about the change.

Reviewer: ```suggestion
Then I should be able to see a text informing me about the change.
```
